source/mesh_deformation/fastscape.cc

```diff
@@ -429,6 +429,11 @@ namespace aspect
           // In later timesteps, we copy h directly from FastScape.
           std::mt19937 random_number_generator(fastscape_seed);
           std::uniform_real_distribution<double> random_distribution(-noise_elevation,noise_elevation);
+          // read sea level from the user defined function or constant value;
+          if (use_sea_level_function)
+            sea_level = sea_level_function.value(Point<1>(time_in_years));
+          else
+            sea_level = sea_level_constant_value;
```

Comment:
If you change this to the following you dont need the `mutable` on the member variable:
```
const double current_sea_level = (use_sea_level_function)
?
sea_level_function.value(Point<1>(time_in_years))
:
sea_level_constant_value;
```

And you can do the same in the other place where you need the sea level.